x-pack/metricbeat/module/statsd/server/data_test.go

```diff
@@ -8,14 +8,616 @@ import (
 	"testing"
 	"time"
 
+	"github.com/elastic/beats/v7/metricbeat/mb"
+
+	"gopkg.in/yaml.v2"
+
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"github.com/elastic/beats/v7/libbeat/common"
 	"github.com/elastic/beats/v7/metricbeat/helper/server"
 	mbtest "github.com/elastic/beats/v7/metricbeat/mb/testing"
+
+	_ "github.com/elastic/beats/v7/libbeat/processors/actions"
 )
 
+func init() {
+	mb.Registry.SetSecondarySource(mb.NewLightModulesSource("../../../module"))
+}
+
+func TestEventMapping(t *testing.T) {
+	mappingsYml := `
+      - metric: '<job_name>_start'
+        labels:
+          - attr: job_name
+            field: job_name
+        value:
+          field: started
```

Comment:
Should the module detect incorrect configurations? For example detect repeated or conflicting fields, such as:
```
      - metric: '<job_name>_<job_status>'
        labels:
          - attr: job_name
            field: job_name
          - attr: job_status
            field: job_name
```
Or:
```
      - metric: '<job_name>_start'
        labels:
          - attr: job_name
            field: job_name
        value:
          job_name: started
```
This could help users preventing mistakes.

Reply:
👍